bot/utils/services.py

```diff
@@ -59,3 +76,5 @@ async def send_to_paste_service(contents: str, *, extension: str = "") -> Option
             f"Got unexpected JSON response from paste service: {response_json}\n"
             f"trying again ({attempt}/{FAILED_REQUEST_ATTEMPTS})."
         )
+
+    raise PasteUploadError("Failed to upload contents to pastebin")
```

Comment:
Pastebin is the name of an actual website, so using it here would be misleading.
```suggestion
    raise PasteUploadError("Failed to upload contents to the paste service")
```